js/queries.js

```diff
@@ -17,8 +19,13 @@ function getSelectedSourceOpts(state) {
   return state.sources.get("selectedSourceOpts");
 }
 
-function getBreakpoints(state) {
-  return state.breakpoints.get("breakpoints");
+function getBreakpoints(state, actor) {
+  if(!actor) {
+    return state.breakpoints.get("breakpoints");
+  }
+  return state.breakpoints.get("breakpoints").filter(bp => {
+    return bp.getIn(["location", "actor"]) === actor;
+  });
```

Comment:
Could this also be two functions: `getBreakpoints` and `getBreakpointsByActor`?

In the case of the editor, I think you'll always want to call `getBreakpointsByActor` even if there isn't a selected actor. If there isn't a selected source, then the editor doesn't want any breakpoints.